src/globemap.ts

```diff
@@ -1511,5 +1516,14 @@ module powerbi.extensibility.visual {
 
             return controlsContainerSVG;
         }
+
+        public static getCategoricalValueByIndex(category: DataViewCategoryColumn, index: number): string {
```

Comment:
This method doesn't return string if value is actually number.
Please use ES2015 template string to convert to string.

Reply:
Fixed